Library/Homebrew/cask/cmd/info.rb

```diff
@@ -46,6 +46,7 @@ def self.get_info(cask)
 
       def self.info(cask)
         puts get_info(cask)
+        Analytics.cask_output(cask)
```

Comment:
```suggestion
        Utils::Analytics.cask_output(cask)
```

and perhaps just require in this file?

Reply:
I tried this and it is not working. Since `Info` is a class

Reply:
Might need to be `::Utils::Analytics`?

Reply:
yes